src/lc0ctl/leela2onnx.cc

```diff
@@ -83,6 +87,10 @@ bool ProcessParameters(OptionsParser* options) {
   options->Add<StringOption>(kOutputValue) = "/output/value";
   options->Add<StringOption>(kOutputMlh) = "/output/mlh";
   options->Add<BoolOption>(kOnnxToPytorch) = false;
+  std::vector<std::string> value_head = {"winner", "q", "st"};
+  options->Add<ChoiceOption>(kValueHead, value_head) = "winner";
```

Comment:
As it's passed to the converter as string anyway, and in the latest version of the proto we may have arbitrary head names, maybe just make it a `StringOption`, and list the "popular" options in the help line?